tests/conftest.py

```diff
@@ -300,6 +301,7 @@ def logged_in_account():
 }
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 297]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->not adding `local_daemon` to unit? to keep up the speed i assume<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/run-house/runhouse/373?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
The vast majority of tests which apply to `local_daemon` as a proper cluster are currently in LOCAL. Putting it in UNIT would mean running a bunch of UNIT tests on a cluster that aren't really used to running on a cluster. I figured this is less disruptive and we can adjust it later if it works fine.